### DISCUSSION TITLE
Offer email users an additional way to to combat email spam: An optional paywall

### DISCUSSION THREAD

**Discussion by SteffenNS** [simple-login/app#1710] [Feature Requests]
Hi Simplelogin

I like your throwaway emails. They can stop some spam, but I don't think they solve all types of spam. If spammers already have your real email address then your current solution is no good, right?

I would like to propose a supplemental way to combat email spam which will also work in the use case mentioned above. I propose that email addresses can be protected by paywalls. With a paywall-protected email address it is no longer free to send spam. It costs, and the email recipient is paid by whoever wants to spam him! With efficient and scalable programmable blockchain payments this is now possible.

But how would a paywall work in relation to email?

I have described my solution on [https://bitcoinsteffen.com/any-email-provider-can-now-stop-spam-efficiently/](https://bitcoinsteffen.com/any-email-provider-can-now-stop-spam-efficiently/). The intended email recipient's email provider automatically sends a payment instruction to the non-whitelisted sender via an auto-reply so the solution is compatible with legacy email providers and legacy email clients, not much different from how a traditional out-of-office reply works. Only if the bill is paid will the email be delivered in the intended recipient's inbox.

Email addresses protected by a paywall will allow the user to have a completely public email without having to worry about spam. With a public email it will also become easier to establish new positive relations among people.

Together with the Swiss blockchain payments experts at [Centi](https://centi.ch/) I can help you implement a solution. 

Let's talk.

Best regards
[Steffen Sølling](https://www.linkedin.com/in/steffen-s%C3%B8lling-180b73103/)
Master of Science (Economics and Business Administration) from Copenhagen Business School, and former Microsoft Certified Systems Engineer (in Windows NT and 2000) and Microsoft Certified Trainer, and self-taught in [Austrian Economics](https://mises.org/what-austrian-economics)
Creator of https://bitcoinsteffen.com/ in 2015
Denmark

**Comment by JoelDalais** [comment]
An addition to this could be a scaling paywall, whereby users can set the amount from a fraction of a penny to much higher. Also, if the email provider felt inclined, they could take 1/1000th of a penny as a fee per email or something similar.

**Reply by SteffenNS** [reply]
Yes, the payment technology is very flexible and allows for both those things to be implemented. You can basically implement what you are able to imagine, so it is really a question of what Simplelogin would like to see implemented. We would gladly have talks with Simplelogin about our and their ideas and/or their possible objections.

**Comment by vjayer** [comment]
Funny you mention this in the context of crypto/btc/bsv because in the proto-crypto history, your proposal and application came up decades ago too first implemented by Adam Back for hashcash
https://en.bitcoin.it/wiki/Hashcash#History

And it keeps coming up. A company a few years ago offered this exact service: Bitbounce who is no longer around.

**Reply by SteffenNS** [reply]
Bitbounce was different from what I propose. 

1) Bitbounce required access to each user's mailbox. My proposal doesn't require access to the user's data.
2) Bitbounce used its own cryptocurrency called credo. My proposal use a general purpose digital cash system which can be used for all sorts of things. A money for each purpose is not how good money works. It is not an efficient system. Good money can be used for all sorts of things.
3) credo couldn't scale and it was expensive to transact on. BSV can scale, and is very cheap to transact on.

Adam Back's HashCash didn't financially reward the email recipient. My proposal does financially reward the email recipient.
